lib/tasks/lae.rake

```diff
@@ -13,10 +13,7 @@ namespace :figgy do
 
     desc "Ingest LAE folders"
     task ingest_disk_files: :environment do
-      folder_dir = ARGV[1]
-      abort "usage: rake lae:ingest_disk_files /path/to/lae/folder" unless folder_dir
-      abort "Error: No such file or directory: #{folder_dir}" unless Dir.exist?(folder_dir)
-      IngestLaeFolderJob.set(queue: :low).perform_later(folder_dir)
+      abort "Use 'Bulk Ingest' under Ephemera to select the folder to bulk ingest instead of using this command."
```

Comment:
```suggestion
      abort "Ephemera folder ingest has been moved into Figgy! Use 'Bulk Ingest' under Ephemera on the home page to select the folder to bulk ingest instead of using this command."
```